playbooks/roles/genesis-exoscale/defaults/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+exo_ubuntu_x64_template: "Linux Ubuntu 16.04 LTS 64-bit 10G Disk (2017-01-28-86454b)"
```

Comment:
Is there an API that can be used to retrieve the latest available template somehow? We try to do this with other providers so that users are launching on an up-to-date image.